recipes/fs/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "fs" %}
+{% set version = "2.0.9" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8570c6349db3b0bf68fe615fe2683ca04796082aceae0ae1d232f71a6f6e47ef" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - appdirs >=1.4.3,<2
+    - pytz
+    - six >=1.10.0,<2
+    - enum34 >=1.1.6,<2  # [py2k]
+    - scandir >=1.5,<2  # [py2k or py34]
+    
+  run:
+    - python
+    - appdirs >=1.4.3,<2
+    - pytz
+    - six >=1.10.0,<2
+    - enum34 >=1.1.6,<2  # [py2k]
+    - scandir >=1.5,<2  # [py2k or py34]
+
+test:
+  requires:
+    - setuptools
+  imports:
+    - fs
```

Comment:
There are a number of imports in the packet; `fs.osfs`, `fs.ftpfs` etc. Do you need to list them here?

Reply:
you can, it's just a test of whether an install succeeded.  If you can import fs you can most likely import fs.osfs etc.  I could tests that subpackages import successfully, but it's not routine to do this in conda-forge.

Reply:
actually the rule may be to test subpackages (where importing can run some code in __init__) but not submodules.